doc/whats_new/v0.23.rst

```diff
@@ -73,6 +73,10 @@ Changelog
   `store_cv_values` is `True`.
   :pr:`15652` by :user:`Jérôme Dockès <jeromedockes>`.
 
+- |Fix| add `best_score_` attribute to :class:`linear_model.RidgeCV` and
+  :class:`linear_model.RidgeClassifierCV`.
+  :pr:`15653` by :user:`Jérôme Dockès <jeromedockes>`.
```

Comment:
Maybe add yourself?

Reply:
It is fine :)

Reply:
it was originally @jeromedockes, I am just splitting the PR such that it is easy to review